### DISCUSSION TITLE
Disable ZoomIn And ZoomOut Instead User can scroll Up/down on mouse wheel and with pressing shift scroll left/right.

### DISCUSSION THREAD

**Discussion by shubh-am8** [xyflow/xyflow#2669] [New Features]
Hi everyone,  i am working on a project with needs this type of functionality using react flow library like i want to disable ZoomIn and ZoomOut feature and need only scroll up/down controlled (means upto defined extent) and to right as much as user wants. [](url)

https://user-images.githubusercontent.com/55014707/206926150-590778f5-9096-443a-9a37-45eb7e634371.mov

**Comment by moklick** [comment]
are you looking for the `panOnScroll` option https://reactflow.dev/docs/api/react-flow-props/#panonscroll ?

**Reply by shubh-am8** [reply]
thanks for sharing the link its really going to help me

**Reply by shubh-am8** [reply]
i have tried this but its not working as per expectation, still zoom in and scroll is working
